src/provider/application.js

```diff
@@ -231,6 +233,13 @@ class Application extends EventEmitter {
   emitError(error) {
     this.emit('xfc.error', error);
   }
+
+  unload() {
+    const nextURL = document.activeElement.href;
+    const target = document.activeElement.target;
```

Comment:
not sure we need target, if the iframe is unloading the target is only `self` isn't it?

Reply:
I guess you're correct - I'll remove it

Reply:
This was added in when it was still part of redirect dictionary. It's how we prevent pages that open in an new tab from also opening in the redirect (by checking for target blank). 

Any idea of a better way to handle this?